### PR TITLE
Bump mlclustering docker image

### PR DESCRIPTION
**Pull request by jlevypaloalto**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Content Pull Request
Related PR: [link to the PR at demisto/content](https://github.com/demisto/content/pull/35421)

## Related Issues
Related: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-11147)

## Description
Bump `mlclustering` docker image libraries to their latest versions.

### CODE REVIEW

File: docker/mlclustering/Pipfile.lock [hunk 1]
```diff
@@ -165,13 +165,16 @@
                 "sha256:873d13d177501a28b2756375d59816c365e42ed8417b41665f346289adc68d24",
                 "sha256:8e5a0b00e1e56a842f922e7fae8ae4077aee4af0acb5ae3622bd4b4c30aedf99",
                 "sha256:8e90497254aacacbc4ea6ae5e7a8cd75629d6ad2b30025a4a8b09aa4faf55151",
+                "sha256:9057e6aa78a584bc93a13f0a9bf7e753a5e9770a30b4d758b8d5f2a62a9433cd",
```
Comment (JudahSchwartz) [inline docker/mlclustering/Pipfile.lock:168]: is every other library already up to date in this image?

**Review by JudahSchwartz — APPROVED** [review]

File: docker/mlclustering/Pipfile.lock (hunk 1 above)
Comment (jlevypaloalto) [reply docker/mlclustering/Pipfile.lock:168]: No, I don't know why `pipenv` is acting this way but most if not all of the libraries were updated.

Comment (JudahSchwartz) [reply docker/mlclustering/Pipfile.lock:168]: so lets run pipenv lock to make sure its up to date

Comment (jlevypaloalto) [reply docker/mlclustering/Pipfile.lock:168]: I did

**Pull request merged**